### PR TITLE
feat: add illustrations #18

### PR DESCRIPTION
**Pull request by jpacsai**
**List of Illustrations included in this PR:**

Solves Issue https://github.com/Anmol-Baranwal/Awesome-Illustrations-4Projects/issues/18

[EMOJIZ](https://emojiz.design/)
[ANIMALZ](https://animalz.design/)
[AVATARZ](https://www.avatarz.design/)
[Nutrition Icons](https://drawkit.com/product/nutrition-icons)
[Mental Health Icons](https://drawkit.com/product/mental-health-icons)
[Website States Icons](https://drawkit.com/product/website-states-icons)
[Education Icons](https://drawkit.com/product/free-education-icons)
[Ecology & Nature Icons](https://drawkit.com/product/ecology-nature-free-icons)
[Startups and SaaS Icons](https://drawkit.com/product/startups-and-saas-icons)
[COVID-19 Icons](https://drawkit.com/product/covid-19)
[Food & Drink Icons](https://drawkit.com/product/food-drink)
[Ugly Delicious](https://drawer.design/products/ugly-delicious/)
[Social Work & Charity](https://drawkit.com/product/social-work-charity-illustrations)
[COVID-19 & Vaccinations](https://drawkit.com/product/covid-19-vaccinations)
[Product & Project Managers](https://drawkit.com/product/project-product-manager-illustrations)

I have used all the change requests for PR #23 (closed that one as merging the upstream fork with new changes has completely messed it up)
---

- [x] I have read the [Contribution Guidelines]

### CODE REVIEW

**Comment by jpacsai** [comment]
@Anmol-Baranwal I have resolved the merge conflict

**Comment by Anmol-Baranwal** [comment]
@jpacsai  
Why is there a gap in the images you have uploaded on the left & right sides?
e.g.
Screenshot (3987)

I am just asking to get to know the reason behind this. Is it your specific configuration or default style?

File: README.md [hunk 1]
```diff
@@ -182,17 +197,22 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
 | <a href="https://craftwork.gumroad.com/l/ZWiR"><img width='700' src='https://user-images.githubusercontent.com/74038190/184810212-6b2ee06f-81a7-413c-b967-a899e9b7b56c.png' alt="Control_free_illustrations" /></a> | [Control](https://craftwork.gumroad.com/l/ZWiR)  <br><br> ✔ Free <br><br>Format <br> • PNG<br> • Adobe Illustrator<br><br> ✌️Author <br> Craftwork.design                                                                                             |
+| <a href="https://drawkit.com/product/covid-19"><img width='700' src='https://user-images.githubusercontent.com/34191321/194713789-cd2508a7-a719-4bb6-90c6-18e6920c6bab.png' alt="COVID-19_icons_free_illustrations" /></a> | [COVID-19 Icons](https://drawkit.com/product/covid-19)  <br><br> ✔ Free <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195566597-f1ce10d9-83bb-4a9f-b514-2ed1195174c0.png` in covid-19 icons

File: README.md [hunk 2]
```diff
@@ -182,17 +197,22 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
+| <a href="https://drawkit.com/product/covid-19-vaccinations"><img width='700' src='https://user-images.githubusercontent.com/34191321/194714608-5619de4e-2d0d-4023-ad47-95eb5be1d299.png' alt="COVID-19_&_Vaccinations_free_illustrations" /></a> | [COVID-19 & Vaccinations Illustrations](https://drawkit.com/product/covid-19-vaccinations)  <br><br> ✔ Free <br><br>Format <br> • Figma<br> • SVG<br> • PNG<br> • AI<br><br> ✌️Author <br> DrawKit |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195567351-f9f769ff-c308-4088-b71a-1617a14b030d.png` in covid-19-vaccinations

File: README.md [hunk 3]
```diff
@@ -182,17 +197,22 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
+| <a href="https://drawkit.com/product/ecology-environment-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193479444-40aa7a75-b8df-4736-a4f0-8001cd362f5b.png' alt="ecology_&_environment_free_illustrations" /></a> | [Ecology & Environment](https://drawkit.com/product/ecology-environment-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • Figma<br><br> ✌️Author <br> DrawKit   |
+| <a href="https://drawkit.com/product/ecology-nature-free-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194713342-0a5264a2-1ebe-4497-9dee-f7136f29033e.png' alt="ecology_&_nature_free_illustrations" /></a> | [Ecology & Nature](https://drawkit.com/product/ecology-nature-free-icons)  <br><br> ✔ Free <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195568112-2d82cf0b-43f8-4e1a-9d0d-bb9628591100.png` in ecology-nature-free-icons

File: README.md [hunk 4]
```diff
@@ -182,17 +197,22 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
+| <a href="https://www.drawkit.com/product/free-education-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194964105-d0659d45-1c5f-4a31-8f29-2c82333232fe.png' alt="education_icons_free_illustrations" /></a> | [Education Icons](https://www.drawkit.com/product/free-education-icons)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit   |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195569147-2bcdf4bc-5fd4-4823-a306-6ef08039cc6d.png` in Education Icons

File: README.md [hunk 5]
```diff
@@ -182,17 +197,22 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
 | <a href="https://craftwork.design/downloads/flow-lava-constructor/"><img width='700' src='https://user-images.githubusercontent.com/74038190/195008950-4f5b442b-c22e-4ff0-9793-ad0edd7231c5.png' alt="Flow_Lava_free_illustrations" /></a> | [Flow Lava](https://craftwork.design/downloads/flow-lava-constructor/)  <br><br> ✔ Free  <br><br>Format <br> • Figma<br> • Sketch<br> • PNG<br> • SVG<br><br> ✌️Author <br> Craftwork |
+| <a href="https://www.drawkit.com/product/food-drink"><img width='700' src='https://user-images.githubusercontent.com/34191321/194964905-fdc1c359-3337-464f-8632-8bf4965af8e5.png' alt="food_&_drink_icons_free_illustrations" /></a> | [Food & Drink Icons](https://www.drawkit.com/product/food-drink)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit   |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195569620-ae39b498-c7f6-4b87-942b-7e6c7a343a50.png`

File: README.md [hunk 6]
```diff
@@ -214,9 +234,11 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
 | <a href="https://github.com/MariaLetta/mega-doodles-pack/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184917535-88553471-26eb-4bf2-a532-b719341a66c8.png' alt="Mega_Doodles_Pack_free" /></a> | [Mega Doodles Pack](https://github.com/MariaLetta/mega-doodles-pack/)  <br><br> ✔ Free <br><br> |
+| <a href="https://www.drawkit.com/product/mental-health-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194963389-3399dedb-e379-462e-890b-bbe449b8d9da.png' alt="Mental_Health_icons_free_illustrations" /></a> | [Mental Health Icons](https://www.drawkit.com/product/mental-health-icons)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit                                                 |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195569982-89003c4d-1116-4aeb-bcb9-8d0b33dbd186.png`

File: README.md [hunk 7]
```diff
@@ -214,9 +234,11 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
 | <a href="https://maryamato88.gumroad.com/l/pbmkt"><img width='700' src='https://user-images.githubusercontent.com/74038190/184478348-653d118b-8797-4566-9a68-cb1fb981b1eb.png' alt="Notion_Style_free_illustrations" /></a> | [Notion-Style](https://maryamato88.gumroad.com/l/pbmkt)  <br><br> ✔ Free <br><br>Format <br> • SVG<br>• PNG<br><br> ✌️Author <br>Mary Amato                                                                      |
+| <a href="https://www.drawkit.com/product/nutrition-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194963072-2a0c777d-e24e-48d7-8156-b185888eb03a.png' alt="nutrition_icons_free_illustrations" /></a> | [Nutrition Icons](https://www.drawkit.com/product/nutrition-icons)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195570876-ffac6d16-8f67-4899-bed2-84cefe6b1378.png`

**Comment by jpacsai** [comment]
@Anmol-Baranwal It's my default style, I have a big screen probably that's the reason

**Comment by Anmol-Baranwal** [comment]
> @Anmol-Baranwal It's my default style, I have a big screen probably that's the reason

Okay, no worries.

File: README.md [hunk 8]
```diff
@@ -226,6 +248,7 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
 | <a href="https://www.getillustrations.com/illustration-pack/product-hunt-illustrations-free-cat-stickers"><img width='700' src='https://user-images.githubusercontent.com/74038190/185657863-a9b730e5-f2cd-41eb-933a-9693d187b761.png' alt="Product_Hunt_free_illustrations" /></a> | [Product Hunt](https://www.getillustrations.com/illustration-pack/product-hunt-illustrations-free-cat-stickers)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PNG<br>• SVG<br><br> ✌️Author <br> Getillustrations                                                       |
+| <a href="https://drawkit.com/product/product-project-managers-illustrations"><img width='700' src='https://user-images.githubusercontent.com/74038190/194872674-c12a20e6-d262-4a0f-b376-e8eb5643435d.png' alt="product_&_project_managers_free_illustrations" /></a> | [Product & Project Managers](https://drawkit.com/product/product-project-managers-illustrations)  <br><br> ✔ Free <br><br>Format <br> • Figma<br> • SVG<br> • PNG<br><br> ✌️Author <br> DrawKit |
```
Comment (Anmol-Baranwal) [inline README.md]: - add a format `AI`

File: README.md [hunk 9]
```diff
@@ -235,13 +258,17 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
+| <a href="https://drawkit.com/product/social-work-charity-illustrations"><img width='700' src='https://user-images.githubusercontent.com/74038190/194873627-62964db3-9a06-4f7f-b184-19fab9d40a64.png' alt="social_work_&_charity_free_illustrations" /></a> | [Social Work & Charity](https://drawkit.com/product/social-work-charity-illustrations)  <br><br> ✔ Free <br><br>Format <br> • Figma<br> • SVG<br> • PNG<br> • AI<br><br> ✌️Author <br> DrawKit     |
+| <a href="https://www.drawkit.com/product/startups-and-saas-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194964526-e758799c-320c-422e-8a16-f27b65b2ce19.png' alt="startups_and_saas_icons_free_illustrations" /></a> | [Startups and SaaS Icons](https://www.drawkit.com/product/startups-and-saas-icons)  <br><br> ✔ Free <br><br>Format <br> • SVG<br> • PNG<br>• JPG<br><br> ✌️Author <br> Drawkit                |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195572891-c6cf167b-bba3-4e17-ab70-326877219f53.png`

File: README.md [hunk 10]
```diff
@@ -235,13 +258,17 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
 | <a href="https://craftwork.design/downloads/tiny-dribbble-shots-constructor"><img width='700' src='https://user-images.githubusercontent.com/34191321/194421585-c93825ae-8d3f-4caf-88ec-0db5e359380d.png' alt="Tiny_Dribbble_Shots_Constructor_free_illustrations" /></a> | [Tiny Dribbble Shots Constructor](https://craftwork.design/downloads/tiny-dribbble-shots-constructor)  <br><br> ✔ Free <br><br>Format <br>• Figma<br><br> ✌️Author <br> Craftwork | 
+| <a href="https://drawer.design/products/ugly-delicious/"><img width='700' src='https://user-images.githubusercontent.com/74038190/194874593-f301ed8b-8c58-4507-aabb-54a826d54f56.png' alt="ugly_delicious_free_illustrations" /></a> | [Ugly Delicious](https://drawer.design/products/ugly-delicious/)  <br><br> ✔ Free <br><br>Format <br> • Sketch<br> • Adobe Illustrator<br> • Adobe Xd<br> • Adobe After Effects<br> • Figma<br> • Lottie<br> • SVG<br> • EPS<br> • PNG<br><br> ✌️Author <br> Drawer |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195573490-a4751388-a786-4b57-b53d-df12c412064b.png` (the size is changed)

File: README.md [hunk 11]
```diff
@@ -235,13 +258,17 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
 | <a href="https://iconscout.com/illustration-pack/valentines-10"><img width='700' src='https://user-images.githubusercontent.com/74038190/184810200-d3adaa2e-8be7-4942-b97a-8bba3d44bb8f.png' alt="Valentine_free_illustrations" /></a> | [Valentine](https://iconscout.com/illustration-pack/valentines-10)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PSD<br>• SVG<br>• AI<br><br> ✌️Author <br> Iconscout       |
+| <a href="https://www.drawkit.com/product/website-states-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194963725-03530fdd-112a-4e11-bcad-10bcc45ccdfd.png' alt="website_states_icons_free_illustrations" /></a> | [Website States Icons](https://www.drawkit.com/product/website-states-icons)  <br><br> ✔ Free <br><br>Format <br> • SVG<br> • PNG<br>• JPG<br><br> ✌️Author <br> Drawkit                                                     |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195574040-c35d2018-cdbd-4d53-b1e8-27c0c730d8e2.png`

File: README.md [hunk 12]
```diff
@@ -253,10 +280,13 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
 | <a href="https://www.abstrakt.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184916241-46cc2949-d749-4695-9813-05569b88af99.png' alt="Abstrakt_freemium_illustrations" /></a> | [Abstrakt](https://www.abstrakt.design/)  <br><br> ✨ Freemium <br><br>Format <br> • PNG<br>• SVG<br><br> ✌️Author <br> Abstrakt.design                                                                                                                |
+| <a href="https://animalz.design/"><img width='700' src='https://user-images.githubusercontent.com/34191321/194711955-ef309dbf-5620-4737-ab62-ff0aabcdd9f0.png' alt="Animalz_freemium_illustrations" /></a> | [Animalz](https://animalz.design/)  <br><br> ✨ Freemium  <br><br>Format <br> • Sketch<br>• Adobe Photoshop<br>• Adobe Illustrator<br>• Adobe XD<br>• Figma<br>• InVision Studio<br>• Blender 3D<br><br> ✌️Author <br> ThreeDee                       |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195574700-bffeaa34-052f-4886-9daa-2d4daf7d43ac.png`

File: README.md [hunk 13]
```diff
@@ -253,10 +280,13 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
 | <a href="https://characterz.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/194710390-b436f918-f494-41a0-bda1-76f205e95937.png' alt="make_this_appropriate" /></a> | [CHARACTERZ 2](https://characterz.design/)  <br><br> ✨ Freemium  <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PSD<br>• AI<br>• PNG<br>• Blender 3D<br><br> ✌️Author <br> ThreeDee                                                            |
+| <a href="https://emojiz.design/"><img width='700' src='https://user-images.githubusercontent.com/34191321/194711655-63359e54-bfee-493d-83e1-3b8b14bb4915.png' alt="Emojiz_freemium_illustrations" /></a> | [Emojiz](https://emojiz.design/)  <br><br> ✨ Freemium  <br><br>Format <br> • Sketch<br>• Adobe Photoshop<br>• Adobe Illustrator<br>• Adobe XD<br>• Figma<br>• InVision Studio<br>• Blender 3D<br><br> ✌️Author <br> ThreeDee                       |
```
Comment (Anmol-Baranwal) [inline README.md]: - change the src link to `https://user-images.githubusercontent.com/74038190/195575740-741a625a-7631-4e4e-9d24-5db0f1f30932.png`

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
Please make these minor changes in a single commit.

And please do not resolve the conflicts since it seems that some illustrations are added again.

File: README.md (hunk 10 above)
Comment (Anmol-Baranwal) [reply README.md]: This was not needed, since your previous one exactly matches my recommended one.

**Review by Anmol-Baranwal — APPROVED** [review]

**Comment by Anmol-Baranwal** [comment]
Appreciate your contribution :)

**Pull request merged**